algo_demo/style.css

```
@@ -156,19 +166,59 @@ main { padding: 32px 0; min-height: calc(100vh - 200px); }
+.viz-filter input:focus { outline: 1px solid var(--text); outline-offset: -2px; }
+.viz-filter input::placeholder { color: var(--text-dim); }
+.viz-filter-count {
+  font-size: 14px; color: var(--text-muted); white-space: nowrap;
+  text-transform: uppercase; letter-spacing: 0.05em;
+}
+.viz-filter-empty { color: var(--text-muted); margin: 24px 0; }
+.algo-card[hidden], .algo-grid[hidden], .section-label[hidden],
+.viz-filter-empty[hidden] { display: none; }
+.visually-hidden {
+  position: absolute; width: 1px; height: 1px; overflow: hidden;
+  clip: rect(0 0 0 0); clip-path: inset(50%); white-space: nowrap;
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)